automem/search/runtime_relations.py

```
@@ -0,0 +1,146 @@
+        MATCH (m:Memory {{id: $id}}){'-[r' + rel_pattern + f'*1..{max_depth}]-' if rel_pattern else f'-[r*1..{max_depth}]-'}(related:Memory)
+        WHERE m.id <> related.id
+        RETURN DISTINCT related
+        ORDER BY coalesce(related.importance, 0.0) DESC, coalesce(related.timestamp, '') DESC
+        LIMIT $limit
+    """
+
+    params = {"id": memory_id, "max_depth": max_depth, "limit": limit}
+
+    try:
+        result = graph.query(query, params)
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)